.github/workflows/ci.yml

```diff
@@ -39,4 +37,67 @@ jobs:
     - name: Lint Python Code
       run: |
             pip install ruff
-            ruff check
\ No newline at end of file
+            ruff check
+
+  demo:
+    name: Demo Job
+    runs-on: ${{ matrix.os }}
+    strategy: 
+      matrix:
+        os: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v5
+      with: 
+        python-version: '3.9'
+        cache: 'pip'
+        cache-dependency-path: setup.py
+    - name: Run CPP Demos
+      run: |
+            python3 -m pip install --user . -v
+            cd examples/KcMwm && echo "KcMwm CPP Demo:"  
+            cmake -B build && cmake --build build && cd build && ./demo
+            cd .. && cd Tsp && echo "Tsp CPP Demo:"
+            cmake -B build && cmake --build build && cd build && ./demo
+            cd .. && cd Ksat && echo "Ksat CPP Demo:" 
+            cmake -B build && cmake --build build && cd build && ./demo
+            echo "CPP Demos executed successfully."
```

Comment:
It appears that not all demos are being printed. So far, I can only see the output of Demo KcMwm printed three times, and the others seem to be missing.

Reply:
Looks like there is a mixup with the directories I'm trying to `cd` into. Fixing this now

Reply:
This should be okay now 👍

Reply:
It is now showing kcmwm once, and the other commands are commented

Reply:
Sorry about that. Should be good again now